remote-config.jsonc

```diff
@@ -74,17 +74,14 @@
         {
           "message": "The FAQ is a useful collection of not-obvious details: https://ddev.readthedocs.io/en/stable/users/usage/faq"
         },
-        {
-          "message": "Platform.sh sponsors DDEV by paying its lead maintainer to work on DDEV full time!"
-        },
         {
           "message": "Need to get the database and files from your Platform.sh stack? Try `ddev pull platform`! https://ddev.readthedocs.io/en/stable/users/providers/platform"
         },
         {
           "message": "Use `ddev composer` instead of `composer` to ensure your project uses the Composer and PHP versions it specifies."
         },
         {
-          "message": "`ddev npm` is the right way to run npm commands in your web container."
+          "message": "`ddev npm` is usually the best way to run npm commands in your web container."
```

Comment:
how about "...is the preferable way..."  instead of "..is usually the best way"?